scripts/deployment-utils.js

```diff
@@ -0,0 +1,25 @@
+const packages = [
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏About this module path: Let's not mix executables and utility files. Maybe move this to `scripts/lib`?

Reply:
why not, I have started another branch to reorganise the scripts directory which has too many files, I'll address this comment there.

Reply:
Sounds good to me